### ISSUE TITLE
This adb server's $ADB_VENDOR_KEYS is not set

### ISSUE THREAD

**Issue by stevenxoyo** [MustardChef/WSABuilds#135]
### Please provide a clear and concise description of the changes you would like to see made to the documentation. | 请清晰简明地描述你希望看到的对文件的修改。

after I installed WSA, all apps like google can be used, and I can install apps from Google play, but when I download *.apk and want to install this apk by wsatools, it shows 
"An error occurred while reading a response from ADB: device unauthorized.
This adb server's $ADB_VENDOR_KEYS is not set
Try 'adb kill-server' if that seems wrong.
Otherwise check for a confirmation dialog on your device."
How to fix this issue, otherwise I only can install apps from google play.

**Comment by infinitepower18** [comment]
Use [WSA Sideloader](https://github.com/MustardChef/WSABuilds/blob/master/Guides/WSA-Sideloader.md). It's way more reliable compared to the others.

**Comment by MustardChef** [comment]
@stevenxoyo I also agree, that [WSA Sideloader](https://github.com/MustardChef/WSABuilds/blob/master/Guides/WSA-Sideloader.md) is the better choice, in this situation.

However, in the case that an issue occurs with WSA Sideloader, please comment on this issue within 3 days (this is when I will close this issue).

**Comment by MustardChef** [comment]
> great, it works fine, thank you very much

Thank you for confirming!
